### PR TITLE
[TOOLS-1748] Progress chart adapted to Highcharts

### PR DESCRIPTION
**Pull request by emannuelalmeida**

### CODE REVIEW

File: src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html [hunk 1]
```diff
@@ -23,11 +23,19 @@
     <template>
 
         <style>
+        	.widget__body {
+            	display: flex;
```
Comment (robersongomes) [inline src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html]: You have mixed tabs and spaces, so the indentation doesn't look correct (specially on gihub).

There are more cases in the rest of the code.

File: src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html [hunk 2]
```diff
@@ -100,158 +106,111 @@
...
+                },
+                
+                _createChartOptions: function() {
+	               var self = this;
+	               this.set('options', [
+	                   {
+	                       icon:'taskboard-icons:dashboard-filter',
+	                       title: 'Chart filters',
+	                       tap: function() {
+	                           self.$.modal.open();
```
Comment (robersongomes) [inline src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html]: You can get rid of `self` if you use arrow function here.

**Comment by emannuelalmeida** [comment]
Reopening as requested.

File: src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html [hunk 3]
```diff
@@ -100,158 +106,111 @@
...
 
-                reset: function() {
-                    this.set('isReady', false);
-                    this.set('errorMessage', '');
+                ready: function () {
+                    this._createChartOptions();
+                },
+                
+                _createChartOptions: function() {
+                    var self = this;
```
Comment (robersongomes) [inline src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html]: I think you forgot to delete this variable

File: src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html [hunk 4]
```diff
@@ -23,11 +23,19 @@
...
 
         <style>
+            .widget__body {
+                display: flex;
+            }
+            #progress-chart {
+                display: flex;
+                height: 100%;
+                width: 100%;
+       	    }
```
Comment (robersongomes) [inline src/main/resources/static/elements/taskboard/kpis/widget-progress-chart.html]: this rule still unindented due to tabs

File: src/main/resources/static/scripts/chart-utils.js [hunk 5]
```diff
@@ -204,3 +197,38 @@ class WeeklyChartBuilder {
...
+                return Highcharts.dateFormat('%e %b %y', this.value);
+            }
+        };
+        
+        this.options.tooltip.xDateFormat = 'Week from %A, %b %e, %Y';
+    }
+}
+
+class ProgressChartBuilder extends ChartBuilderBase {
+	constructor(divID, startDate, endDate){
```
Comment (robersongomes) [inline src/main/resources/static/scripts/chart-utils.js]: this whole constructor is unindented due to tabs

**Pull request merged**